aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,63 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
 
-.. note::
+ The follwoing instrcution were tested using CentOS 7.
+ 
+Install Mono
+^^^^^^^^^^^^
+
+`Mono <http://mono-project.com>`_ is an ongoing effort to port the .NET Framework to other platforms. Mono is one of the ways .NET applications can run on platforms other than Windows. ASP.NET 5 requires a version of Mono greater than 4.0.1.
+
+To install Mono::
+
+First import the public key from ubuntu.com::
+
+    sudo rpm --import "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF"
+
+Now your CentOS 7 have the public key from this server.
+
+In the next Step we want add the repo from the mono project. For this step we need a tool collection called yum-utils.
+Wie install it with this command::
+
+    sudo yum install yum-utils
+
+Now we can add the repositorie from mono::
+
+    sudo yum-config-manager --add-repo http://download.mono-project.com/repo/centos
+    sudo yum update
+
+In the last step we can now install mono::
+
+    sudo yum install mono-complete
 
-    Installation steps for CentOS, Fedora and derivatives are not currently available but should be available soon. The commands are mostly the same, with some differences to account for the different package managers used on these systems. Learn how you can `contribute <https://github.com/aspnet/Docs/blob/master/CONTRIBUTING.md>`_ on GitHub.
-  
+Install Libuv
+^^^^^^^^^^^^^
+
+`Libuv <https://github.com/libuv/libuv>`_ is a multi-platform asynchronous IO library that is used by the `KestrelHttpServer <https://github.com/aspnet/KestrelHttpServer>`_ that we will use to host our ASP.NET 5 web applications.
+
+To build libuv you should do the following::
+
+    sudo yum install automake libtool wget
+    wget http://dist.libuv.org/dist/v1.6.1/libuv-v1.6.1.tar.gz
+    tar -zxf libuv-v1.6.1.tar.gz
+    cd libuv-v1.6.1
+    sudo sh autogen.sh
+    sudo ./configure
+    sudo make
+    sudo make check
+    sudo make install
+    ln -s /usr/lib64/libdl.so.2 /usr/lib64/libdl
+    ln -s /usr/local/lib/libuv.so /usr/lib64/libuv.so.1
+
+
+Install the .NET Version Manager (DNVM)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Now let's get DNVM. To do this run::
+
+    curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh && source ~/.dnx/dnvm/dnvm.sh
+    sudo chmod +x ~/.dnx/dnvm/dnvm.sh
+    ~/.dnx/dnvm/dnvm.sh
```

Comment:
Just tested this, the last 2 lines are not needed
We might want to mention putting source ~/.dnx/dnvm/dnvm.sh in the .bash_profile or .bashrc